src/components/WalletNameStep/WalletNameStepContainer.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+import { pure } from 'recompose'
+
+import WalletNameStep from './WalletNameStep'
+
+export default pure(WalletNameStep)
```

Comment:
Here, there will be problems with flowtype. All prop types will be unavailable in another files. After discussion we decided to use React.PureComponent instead of recompose/pure

Reply:
https://github.com/jibrelnetwork/jwallet-web/blob/feature/update-import-wallet/src/components/WalletNameStep/WalletNameStep.js

Reply:
just functional component